script/Arcueid/Day 6/06_00_ARC06_2.txt

```diff
@@ -56,7 +59,8 @@ Well... no, but actually...
 [sha:fafe7120f9c25faf6608ffadfb541941b4a2eb15]{
 -- Page 139, Offset 14261.
 -- 携帯はー……こっそりー……返してもらっていたとー……言うかー……。
-My cell phone... has alredy been surreptitiously returned...
+// didn't we use mobile phone in days 1-5?
```

Comment:
Looks like cell phone might be the Amerihiko version.

Reply:
Should we go back change the ones in day 1 then?
https://github.com/Tsukihimates/Tsukihime-Translation/blob/b6a3f8bd2a6299d03885d8f419c183ebce9e3fdf/script/Arcueid/Day%201/01_00_ARC01_9.txt#L240